document/js-api/index.bs

```diff
@@ -765,6 +766,8 @@ Each {{Table}} object has a \[[Table]] internal slot, which is a [=table address
 <div algorithm>
     The <dfn constructor for="Table">Table(|descriptor|, |value|)</dfn> constructor, when invoked, performs the following steps:
     1. Let |elementType| be [=ToValueType=](|descriptor|["element"]).
+    1. If |elementType| does not match [=reftype=],
```

Comment:
```suggestion
    1. If |elementType| is not a [=reftype=],
```
(`[=reftype=]` is not a concrete type that could be matched. Nor do we define what "match" means.)